staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -388,10 +388,23 @@ func (s *GenericAPIServer) DynamicApisDiscovery() *restful.WebService {
 		// sort to have a deterministic order
 		sortedGroups := []metav1.APIGroup{}
 		groupNames := make([]string, 0, len(s.apiGroupsForDiscovery))
+
+		// hack to move apps after extensions so that the Deployment resources is found by
+		// kubectl in 1.5
+		foundApps := false
```

Comment:
Are there other resources shared between extensions and apps this will reverse from the desired order?